### ISSUE TITLE
Steps for Inferencing on Custom Dataset 

### ISSUE THREAD

**Issue by alpharajat** [punyajoy/HateXplain#13]
Hi team,
I couldn't find any API/Library that can help me run me prepare the dataset and provides inferences for all the Main(3) & Hierarchical Classes(8) using the already trained model parameters.
Can you help with a steps so that I can get the inferences without any training and just inferencing on Test Data directly,
(This idea is inspired from the Detoxify library for Unitary Model in Hugging Face)

**Comment by punyajoy** [comment]
Please follow this https://huggingface.co/Hate-speech-CNERG/bert-base-uncased-hatexplain-rationale-two

**Comment by alpharajat** [comment]
Thank You for helping with the required stuff.
My dataset has only text column for which I have found the Main Class output(2-Normal, Abusive). But I am not able to interpret the 2- Class model output apart from the prediction_logits.  Please look into the attached SS.
I am able to get output as JSON
`[[{'label': 'NORMAL', 'score': 0.4873974323272705},
  {'label': 'ABUSIVE', 'score': 0.5126025676727295}]]`
<img width="680" alt="HateExplain" src="https://user-images.githubusercontent.com/90767471/162404503-ab18387c-346f-4363-a102-8528ccc6e24d.PNG">


Also, I am able to generate the output for 3 Class (3-Hate speech, Offensive language, None) output, using the code-
`[[{'label': 'hate speech', 'score': 0.0430845282971859},
  {'label': 'normal', 'score': 0.709394633769989},
  {'label': 'offensive', 'score': 0.2475208044052124}]]`

But , I am struggling to get the explainable Output in terms of target( Who are the targets?) for both 2 Class and 3 Class labels, for the custom dataset
for example Label = Hate
                    Target = African

**Comment by punyajoy** [comment]
We still do not support the target classification since the classification performance is still not satisfactory
